src/main/java/uk/gov/companieshouse/api/accounts/model/ixbrl/balancesheet/BalanceSheetStatements.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.companieshouse.api.accounts.model.ixbrl.balancesheet;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class BalanceSheetStatements {
+
+    @JsonProperty("section477")
+    private String section477;
+    @JsonProperty("audit_not_required_by_members")
+    private String auditNotRequiredByMembers;
+    @JsonProperty("directors_responsibility")
+    private String directorsResponsibility;
+    @JsonProperty("small_companies_regime")
+    private String smallCompaniesRegime;
+
+    
```

Comment:
Minor alteration, either have 2 spaces or 1. Bit of inconsistency with spacing on some of the classes